### ISSUE TITLE
Find Topic / Concept

### ISSUE THREAD

**Issue by vogelino** [FH-Potsdam/bit-nickel-weasel#2]
Form asana by @wollemannone: Frameconditions for all groups

**Comment by vogelino** [comment]
One Idea would be to force all groups to develop a modular ball parkour that is able to receive a ball at one precise __input position__, do some fancy stuff with it and finally retrieves it at a a final __output position__ (that represent the __input position__ of the next object). That way, all automated object could be chained in any position. This concept could be extended with any dimension: instead of a ball, this could be a liquid, sand, candy, whatever…
@FH-Potsdam/connecting-bits-nickel-weasel what do you think?

**Comment by vogelino** [comment]
The problem is that object cannot be distant to each other. Also, the use of wifi would be only optional, as sensor could be use to detect input and output. :(

**Comment by vogelino** [comment]
We are four groups. That means that we could distributes the following groups of four topics:
- Spring, summer, autumn and winter
- Water, wind, fire, earth
- Sanguine (optimistic and social), choleric (short-tempered or irritable), melancholic (analytical and quiet), and phlegmatic (relaxed and peaceful) :star: :star: :star: :star: :star: Because their original purpose is to define humans and in our case we would define machines… 
- The four musketeer
- Etc.

**Comment by vogelino** [comment]
@fabiantheblind Do we need a keynote presentation for tomorrow or would it be more a discussion? Thx!

**Comment by fabiantheblind** [comment]
If you like put together a keynote. It is easier to shoe your ideas with some images

**Comment by vogelino** [comment]
Right. I put our ideas together in a keynote then. Should we present our absolute best idea or is it okay to come with three or four different ideas and concepts? Thx!

**Comment by fabiantheblind** [comment]
Lets brainstorm first. We will have to poll or one idea from all the groups maybe we can combine some stuff

**Comment by vogelino** [comment]
Got ya. Thx!

**Comment by wollemannone** [comment]
uups
 hi there, I like the first idea. If i get it right, there is also a physical interaction through the ball besides the signal start/stop.

Here comes the idea I prepared. Unfortunatly I´m not gonna be there tomorrow. But I hope it helps a bit.

![conceptidea_connecting bits_wolfgang](https://cloud.githubusercontent.com/assets/15340581/12120031/3a72b380-b3cf-11e5-9d21-467dbb5d583f.jpg)

**Comment by wollemannone** [comment]
Maybe it would be good to combine the box idea with the seasons - theme.
Every group creates a box, following the the frame conditions. The actions and the medium is according a certain season.

I´m excited to hear what we´re gonna do. Wish you good luck tomorrow.

**Comment by vogelino** [comment]
Nice! Thank you for your work! I will try to summarize it in the presentation and to look for ways to combine ideas. I'm on it!

**Comment by vogelino** [comment]
@dariaDunkelbier could you fill the presentation with images and drawings? I created a sketch document already but it would be great if you could terminate it. I wrote titles that represent the many ideas that @wollemannone you and me had. If one is not clear just get me a call. Okay?

Here is the sketch document, the same version exported in .ai and the font i used for this. Feel free to change it and let me know if you have no time for it. Thx!
[IO-Ideas-Presentation-6.1.16.zip](https://github.com/FH-Potsdam/bit-nickel-weasel/files/78796/IO-Ideas-Presentation-6.1.16.zip)

**Comment by wollemannone** [comment]
hey guys, 
how was it today? Did you get ready with the presentation? How is the conclusion of today?

**Comment by vogelino** [comment]
hi @wollemannone!

Unfortunately we couldn't agree on one topic. The idea of the four temperaments seemed to appeal most of us. Also the idea of creating machines that act based on moral principle is an option. Another idea that most of us liked is using similar looking black boxes that have different behavior or functionalities.

He heavily discussed about wether the audience takes part of the interaction (human to machine) or wether the asynchronous execution of the machines/objects are a good idea. 

However, the fact that we based most of our concepts on factual aspects revealed the lack of meaning of our propositions. That's why we agreed about delaying the decision to the next course. For the following Wednesday, every team has to come up with a final concept that can wether be:
- based on a research question (thesis)
- based on the idea of telling a story
- based on another statement

The team responsibilities were defined but we still miss a responsible person for the online documentation. I created an [issue](https://github.com/FH-Potsdam/bit-nickel-weasel/issues/8) for that.

I wanted to take part of the workshop tomorrow and worked on my free Monday especially for it, but I feel sick and won't be present. I do not remember if you will be there. @dariaDunkelbier will take notes but if you will be present try to write your own also. That would be nice! ;)

If you have any additional question, don't hesitate. See you next week then!

**Comment by wollemannone** [comment]
Okay thanks very much for keeping me updated. Unfortunately I´ll be only back on Saturday so we depend on @dariaDunkelbier s notes. But I bet they will be good to work with :). 
I was wondering if we should have a relaxed meeting to talk about our concept or develop it together? What would you say?
@dariaDunkelbier  nice drawings in the presentation. Thank you

**Comment by vogelino** [comment]
Okay no problem. I'm sure @dariaDunkelbier 's notes will be sufficient too. It would be a good idea to make a @FH-Potsdam/connecting-bits-nickel-weasel meeting until next Wednesday. When would you have time? On Saturday 14h? We can do it at my place if you wish. I have no time on Sunday and Saturday night but am free on Monday too if you prefer. Let me know!

**Comment by wollemannone** [comment]
I´ve a course on Saturday so Monday would be better for me. What about Daria? Did she ever respond in the github chatroom?

**Comment by vogelino** [comment]
No I never saw a message from her but I told her to look at it and she said she would... I'll text her on her phone then. PS: Monday would be good for me too.

**Comment by wollemannone** [comment]
Okay let´s say Monday than. I think it´s enough to have a brief meeting just to confirm our ideas and to distribute the tasks. Do you think about 10:00 am would be a good time?

**Comment by dariaDunkelbier** [comment]
monday is totally fine
where

**Comment by dariaDunkelbier** [comment]
10 is perfect

**Comment by dariaDunkelbier** [comment]
i did a lot of notes for you!! i hope i understand them still on monday =) but we will figure it out together

**Comment by vogelino** [comment]
We could do it at my place if you want. Forster Straße 2, U Görli, Bus Spreewaldplatz. 14 Uhr. okj?

**Comment by wollemannone** [comment]
All right, lets meet on Monday. I would prefer to meet more early but if not possible 14 is okay for me.

**Comment by dariaDunkelbier** [comment]
i have to work at 15.. why not 10??

**Comment by dariaDunkelbier** [comment]
i send the money to you @vogelino

**Comment by dariaDunkelbier** [comment]
danke fürs bestellen =) 
![tahnks](https://cloud.githubusercontent.com/assets/15330337/12222338/e4ca0c14-b7b9-11e5-8b05-37b1ebb75d13.gif)

**Comment by wollemannone** [comment]
yes lets say 10... Lukas?

**Comment by dariaDunkelbier** [comment]
would be perfect for me

**Comment by vogelino** [comment]
How would it be at 12? I have to work on another project before. Or later
in the evening? Sorry about that. @dariaDunkelbier: Thx for the transfert!

2016-01-10 17:05 GMT+01:00 dariaDunkelbier <notifications@github.com>:

**Comment by dariaDunkelbier** [comment]
ok 12 is good

**Comment by dariaDunkelbier** [comment]
so at your place at 12

**Comment by wollemannone** [comment]
check :) see you than

**Comment by vogelino** [comment]
Perfekt! Se you tomorrow!

**Comment by vogelino** [comment]
Hey @FH-Potsdam/connecting-bits-nickel-weasel, I have heavy headache again and won't go to school today either so we won't meet on midday. If you need everything from me let me know. We could also meet via skype and discuss the remaining tasks if you wish. I'm still reachable as always so let me know! Sorry about that!

**Comment by dariaDunkelbier** [comment]
alright..get well soon

**Comment by wollemannone** [comment]
@vogelino just transfered the money

**Comment by vogelino** [comment]
Okay thx!

**Comment by vogelino** [comment]
@dariaDunkelbier I received your money. Thx! :smiley:
